src/jq_port.erl

```diff
@@ -202,22 +206,38 @@ kill_port(Port) ->
         {Port, {data, <<"exiting">>}} -> 
             ok
     after TimeToWaitForExitingConfirmation ->
+              {os_pid, OsPid} = erlang:port_info(Port, os_pid),
```

Comment:
maybe handle `undefined`? Port might be dead already?

Reply:
I think this is handled: https://github.com/emqx/jq/pull/27/files/42cb67c8371c16dcc21486869c33be2ceb047a58#diff-86e26a1935fd813f6a4d0d34b70e647880565f0162f867be8de7d8e535454452R228

Reply:
Fixed in [4d91ca5](https://github.com/emqx/jq/pull/27/commits/4d91ca57a35c8039bc336422b15482bd15312cfa)